performance/README.md

```diff
@@ -0,0 +1,59 @@
+## Setup
+To run locally first install k6. The browser module is now included in the default install(https://github.com/grafana/k6)
```

Comment:
just make this `To run locally first install [k6](https://github.com/grafana/k6). The browser module is now included in the default install.`